gradle/dependencies.gradle

```diff
@@ -21,7 +21,7 @@ def versions = [
     dokka: '0.9.17',
     errorProne: '2.3.1',
     errorPronePlugin: '0.0.13',
-    kotlin: '1.2.21',
+    kotlin: '1.2.51',
     support: '27.0.2'
```

Comment:
nit: doesn't match the version in sample.gradle.

Reply:
Good catch - 731aa55